octobot_trading/exchanges/connectors/ccxt/ccxt_websocket_connector.py

```diff
@@ -39,6 +40,7 @@
 class CCXTWebsocketConnector(abstract_websocket.AbstractWebsocketExchange):
     INIT_REQUIRING_EXCHANGE_FEEDS = [Feeds.CANDLE]
     SUPPORTS_LIVE_PAIR_ADDITION = True
+    FEED_INITIALIZATION_TIMEOUT = 15 * commons_constants.MINUTE_TO_SECONDS
```

Comment:
So if we can't fetch the exchange candle histoiry we wait 15min right ? It's an edge case but I don't know if we should wait so much time